packages/blockchain-extension/extension/commands/environmentConnectCommand.ts

```diff
@@ -53,6 +53,7 @@ export async function fabricEnvironmentConnect(fabricEnvironmentRegistryEntry: F
                     // failed to start local fabric so return
                     return;
                 }
+                await ExtensionUtil.sleep(250);
```

Comment:
why do we need to sleep here?

Reply:
Need this to make sure the Fabric is started before connected.